src/main/java/rs117/hd/scene/model_overrides/WindDisplacement.java

```diff
@@ -3,6 +3,7 @@
 public enum WindDisplacement {
 	DISABLED,
 	OBJECT,
+	OBJECT_NO_GROUND_DISPLACEMENT,
```

Comment:
This has no matching definition in `comp_common.glsl/cl`, and as far as I can tell it wasn't accidentally stripped in a merge, though I could be wrong. Since I'm not quite sure what the intent was here, I'll leave it up to you :see_no_evil: